src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+public class Task {
+   /** Task description */
+    private String task;
+    /** Task status */
+   private boolean taskIsDone = false;
+   public Task(String line){

```

Comment:
There are inconsistency in the number of spaces here. Please follow the coding standard.